modules/update-cluster-version-object.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-cluster-version-object_{context}"]
+= The ClusterVersion object
+
+One of the resources that the Cluster Version Operator (CVO) monitors is `ClusterVersion`.
+
+Administrators and OpenShift components can communicate or interact with the CVO through the `ClusterVersion` object.
+The desired CVO state is declared through the `ClusterVersion` object and the current CVO state is reflected in the object's status.
+
+[NOTE]
+====
+Do not directly modify the `ClusterVersion` object. Instead, use interfaces such as the `oc` CLI or the web console to declare your desired update target.
+====
+
+The CVO continually reconciles the cluster with the target state declared in `ClusterVersion` `spec`.
+When the desired release differs from the current one, that reconciliation updates the cluster.
+
+//to-do: this might be heading overload, consider deleting this heading if the context switch from the previous paragraph to this content is smooth enough to not require one.
+[discrete]
+== Update availability data
+
+The `ClusterVersion` resource also contains information about updates that are available to the cluster.
+This includes updates that available but not recommended due to a known risk that applies to the cluster, which are known as conditional updates.
+To learn about how the CVO maintains this information about available updates in the `ClusterVersion` resource, see the "Evaluation of update availability" section.
+
+You can inspect all available updates with the following command:
+
+[source,terminal]
+----
+$ oc adm upgrade --include-not-recommended
+----
+
+The additional `--include-not-recommended` parameter includes updates that are available but not recommended due to a known risk that applies to the cluster.
```

Comment:
Wondering if this would work better as a IMPORTANT block, but WDYT?

Edit: Might not be needed as a important block if you reformat the commands and outputs